resources/local/config.toml

```diff
@@ -37,6 +37,8 @@ secret_key_path = 'secret_key.pem'
 # The folder in which the files with per-era latest unit hashes will be stored.
 unit_hashes_folder = "../node-storage"
 
+# The duration for which incoming vertices with missing dependencies should be kept in a queue.
+pending_vertex_timeout = '1day'
```

Comment:
Really, that long? Even if eras are supposed to be 30minutes?

Reply:
That's a good point.
Do we have test data on how long it will take to fully sync a complete era's protocol state?
Should I just make it `'30min'`?

Reply:
Why not use `era_duration` for that?

Reply:
Last week we had discussed that we'd make it configurable.
I don't think it should always just be the `era_duration`. It would be better if we could just measure how long syncing a full era takes in practice, and use that value.